src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -11,9 +11,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
```

Comment:
this happened to both me and Liyah, not sure what's up with it, but you can get rid of all these spaces (like the character space " " not the line) on lines 14, 16, 31, 35, 73, 77, 82, 86, 90, 115, 120, 133, 139, 144, 163